float64 x
float64 y
float64 z
bool tool_vertical
bool cartesian_interp
float64 move_time
---
bool success
